### ISSUE TITLE
'Useless assignment' reported when setting atttribute on method parameter

### ISSUE THREAD

**Issue by dirkbolte** [rubocop/rubocop#438]
The following code results in a warning: `Useless assignment to local variable test_class.` for `test_class.member = value`:

``` ruby
# encoding: UTF-8
# TestClass
class TestClass
  attr_accessor :member
end

def add_to_hash(test_class, value)
  test_class.member = value
end
tc = TestClass.new
add_to_hash(tc, 'a')
puts tc.member
```
